clients/drcachesim/CMakeLists.txt

```diff
@@ -296,6 +296,10 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR}/tracer)
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}/scheduler)
```

Comment:
> There are a bunch of PT repos on Github: are they able to automate their tests?

I found a few but they didn't have Github actions configured. Can you share links for the ones you found?

Are these actually collecting traces using PT (which requires root and the PT feature on the processor) or just doing decoding using libipt?

>  Or maybe they set up their own-hosted runner -- if so, we should look into that as it would be best to have automated testing.

If we don't find anything I'll create a new issue (separate from #5505) for this.

Reply:
Also: I think we considered having a checked-in PT trace so that we could at least have automated tests for the raw2trace part, but that would require checking-in the kcore too which can be ~100MB.

Reply:
I only did a shallow search so I don't think I saw anything you wouldn't see in a quick search.

For the manual tests you ran: if none of those have memtrace static-linked into the app, could we add such a test, and confirm that it fails on the malloc import issue?

Reply:
I added -enable_kernel_tracing to the client options in burst_static but it didn't fail.

Reply:
As updated in other comment: I was able to get the existing kernel.simple test fail by adding `-raw_compress none`. So we're testing the use of unsafe functions now.